core/src/poh_service.rs

```diff
@@ -27,12 +28,16 @@ impl PohService {
     ) -> Self {
         let poh_exit_ = poh_exit.clone();
         let poh_config = poh_config.clone();
+        let core_ids = core_affinity::get_core_ids();
```

Comment:
any reason to not move this line right above line 38?

Reply:
no reasons